zipkin-receiver-kafka/src/main/scala/com/twitter/zipkin/receiver/kafka/KafkaStreamProcessor.scala

```diff
@@ -1,28 +1,29 @@
 package com.twitter.zipkin.receiver.kafka
 
 import com.twitter.logging.Logger
-import kafka.consumer.KafkaStream
-import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
 import com.twitter.util.{Await, Future}
+import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
+import kafka.consumer.KafkaStream
 
-case class KafkaStreamProcessor(
-  stream: KafkaStream[Option[List[ThriftSpan]], Option[List[ThriftSpan]]],
+case class KafkaStreamProcessor[T](
+  stream: KafkaStream[T, Option[List[ThriftSpan]]],
```

Comment:
it's so strange that this requires an Option of a List of a Span. I suppose if you don't have control of the List creation (where you could get a null) then having Option[null] makes sense. But, I don't see that happening.
What happens when you remove the Option?